### ISSUE TITLE
Build conda packages in all platforms

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#34]
It would be nice to have an automated process where only the release versions are uploaded to my binstar channel.

Also, reflect changes in installation process in the README.

**Comment by Juanlu001** [comment]
Interesting links:
* http://docs.travis-ci.com/user/deployment/releases/
* http://conda.pydata.org/docs/travis.html
* http://stackoverflow.com/questions/28217556/travis-ci-auto-tag-build-for-github-release

**Comment by Juanlu001** [comment]
Cannot yet provide OS X testing, see #44.

**Comment by Juanlu001** [comment]
Cannot do `conda convert poliastro.tar.bz2 --platform ANY` until conda/conda-build#377 is released, unless I patch conda-build on the fly.

**Comment by Juanlu001** [comment]
Now `conda convert` works, but the conversion process has to be done *after* the packages are built or put the converted version in a different directory.

**Comment by Juanlu001** [comment]
The solution suggested by @newlawrence to test a package on Linux, OS X and Windows is to use Travis CI for OS X and anaconda.org for Linux.

http://docs.anaconda.org/examples.html#AnacondaOrgBuild

I will try it and see how it goes! :)

**Comment by Juanlu001** [comment]
Closing this after #67 is merged. Thanks @newlawrence!

**Comment by newlawrence** [comment]
You're welcome! 😊
